tasks/main.yml

```diff
@@ -110,7 +112,7 @@
       # do if the app is not enabled and ( (archive path is not "") or (app is disabled) )
       when:
         - item.key not in nc_available_apps.enabled
-        - (item.value is not none) or (item.key in nc_available_apps.disabled)
+        #- (item.value is not none) or (item.key in nc_available_apps.disabled)
```

Comment:
Commented out code should be removed or used if still needed.

Reply:
Done